apps/crn-frontend/src/shared-research/state.ts

```diff
@@ -160,3 +163,11 @@ export const useSetResearchOutputItem = () => {
     setRefresh(refresh + 1);
   };
 };
+
+export const useUserPerrmisions = (
```

Comment:
```suggestion
export const useUserPermissions = (
```

Reply:
that typo tho, I have to find the spelling vsCode plugin Tom told me about